### PR TITLE
Accessibility review fixes

### PR DESCRIPTION
**Pull request by jaortiz**

### CODE REVIEW

File: components/pagination/examples/00-default.js [hunk 1]
```diff
@@ -46,13 +46,11 @@ function Example({ brand }) {
 				back={{
 					visible: true,
 					text: 'Go back',
-					assistiveText: page => `Go to previous page which is ${page}`,
```
Comment (dominikwilkowski) [inline components/pagination/examples/00-default.js]: Can we not remove the examples? Rather let's make them better or people don't know about the API?!

Comment (jaortiz) [reply components/pagination/examples/00-default.js]: In the intopia review notes, they said that the text used for the aria-label in the back/next buttons should be matching with the actual button text. So I changed the code to reflect this and therefore the assistiveText option isnt being used anymore.

Comment (dominikwilkowski) [reply components/pagination/examples/00-default.js]: oh... I see.. we removed it completely? ok cool ... I guess

File: components/pagination/examples/00-default.js [hunk 2]
```diff
@@ -47,11 +47,12 @@ function Example({ brand }) {
 					visible: true,
 					text: 'Go back',
 					onClick: (event, page) => console.log(`Go to ${page}`, event),
+					assistiveText: page => `Go to previous page which is ${page}`,
 				}}
 				next={{
 					visible: true,
 					text: 'Go forth',
-					onClick: (event, page) => console.log(`Go to ${page}`, event),
```
Comment (dominikwilkowski) [inline components/pagination/examples/00-default.js]: Why remove the `onClick`?

Comment (jaortiz) [reply components/pagination/examples/00-default.js]: That was an accident. Added it back

Comment (dominikwilkowski) [reply components/pagination/examples/00-default.js]: easy!

**Pull request merged**